project/src/components/reviews-list/reviews-list.tsx

```diff
@@ -0,0 +1,26 @@
+import ReviewItem from '../review-item/review-item';
+import {Review} from '../../types/review';
+
+type reviewsProps = {
+  reviews: Review[],
+}
+
+function ReviewsList({reviews}: reviewsProps): JSX.Element {
+  return(
+    <>
+      <h2 className="reviews__title">Reviews &middot; <span className="reviews__amount">{(reviews) ? reviews.length : 0}</span></h2>
```

Comment:
```suggestion
      <h2 className="reviews__title">Reviews &middot; 
      <span className="reviews__amount">
      {reviews.length}
      </span>
      </h2>
```

ну и пробелы расставить

Reply:
Не исправлено